site/src/style/theme-d.ts

```diff
@@ -17,15 +27,11 @@ declare module '@mui/material/styles' {
     };
     misc: {
       midGray: string;
+      success: TwoToneColor;
+      error: string;
```

Comment:
To my knowledge, success and error in most themes are defined at the top level, and I think that should be the case here as well (as opposed to putting it inside `misc`).